pkg/grpc/proxy/director.go

```diff
@@ -4,7 +4,8 @@
 package proxy
 
 import (
-	"golang.org/x/net/context"
+	"context"
```

Comment:
Reviewers: removed the old, deprecated `context` package and use the standard library (the old package was added as a denied one to the linter)